package.json

```
@@ -46,6 +46,7 @@
     "typecheck": "tsc --incremental --noEmit --skipLibCheck"
   },
   "dependencies": {
+    "@heroicons/react": "^2.0.18",
```

will need to pin this dep

```suggestion
    "@heroicons/react": "2.0.18",
```

should we go ahead and remove lucide completely now?